.azure_pipeline.yml

```diff
@@ -3,21 +3,64 @@ jobs:
 
 - template: continuous_integration/azure/windows.yml
```

Comment:
I think you can get rid of the template and put the job declaration directly at the top level along with global environment variables and the steps.

Reply:
Good idea.